test/compile/scale.test.ts

```diff
@@ -389,11 +389,51 @@ describe('Scale', function() {
   });
 
   describe('rangeMixins()', function() {
-    // FIXME
+    describe('row', function() {
+      // TODO:
+    });
+
+    describe('column', function() {
+      // TODO:
+    });
+
+    describe('x', function() {
+      // TODO: X
+    });
+
+    describe('y', function() {
```

Comment:
Oh I see. That case is `size`'s scale, not `x`, `y` scale, and thus should be in line 422.